sonic-ledd/scripts/ledd

```diff
@@ -188,38 +186,40 @@ def main():
     signal.signal(signal.SIGINT, signal_handler)
     signal.signal(signal.SIGTERM, signal_handler)
 
-    # Load platform-specific LedControl class
-    err = load_platform_led_control_module()
-    if err != 0:
+    # Load platform-specific LedControl module
+    led_control = load_platform_led_control_module()
+    if led_control is None:
         sys.exit(1)
 
-    # Connect to APPL_DB using SwSS SDK
-    swss = swsssdk.SonicV2Connector()
-    swss.connect(swss.APPL_DB)
+    # Open a handle to the Application database
+    appl_db = swsscommon.DBConnector(swsscommon.APPL_DB,
+                                     REDIS_HOSTNAME,
+                                     REDIS_PORT,
+                                     REDIS_TIMEOUT_USECS)
 
-    # Loop forever
-    while True:
-        # TODO: Subscribe for notifications from DB when ports come or go. Add/remove
-        # dictionary entries as necessary.
+    # Subscribe to PORT table notifications in the Application DB
+    sel = swsscommon.Select()
+    sst = swsscommon.SubscriberStateTable(appl_db, swsscommon.APP_PORT_TABLE_NAME)
+    sel.addSelectable(sst)
 
-        # Get a list of all ports from the database
-        port_table_keys = swss.keys(swss.APPL_DB, PORT_TABLE_PREFIX + '*', blocking = True);
+    # Listen indefinitely for changes to the PORT table in the Application DB
+    while True:
+        (state, c, fd) = sel.select()
```

Comment:
Can you please change it to  (state, c) = sel.select()
The interface was changed just recently.

Reply:
Done. Thanks for the heads-up, Pavel!